gravitee-apim-portal-webui-next/src/app/applications/application/application-tab-logs/application-log-table/application-log-table.component.ts

```diff
@@ -187,35 +187,26 @@ export class ApplicationLogTableComponent implements OnInit {
       tap(values => this.initializeFiltersAndPagination(values)),
       switchMap(values => {
         const from = this.computeStartDateTimeFromQueryParams(values.from, values.to, values.period);
-        return this.applicationLogService.list(this.application.id, { ...values, from }).pipe(
-          catchError(err => {
-            console.error(err);
-            return of({
-              data: [],
-              metadata: {
-                data: {
-                  total: 0,
-                },
-              },
-            } as LogsResponse);
-          }),
-        );
+        return this.applicationLogService.list(this.application.id, { ...values, from });
       }),
-      tap(response => {
-        this.totalLogs.set((response.metadata['data'] as LogsResponseMetadataTotalData).total);
+      tap(({ metadata }) => {
+        this.totalLogs.set((metadata['data'] as LogsResponseMetadataTotalData).total);
       }),
-      map(response =>
-        response.data.map(log => ({
+      map(response => ({
+        data: response.data.map(log => ({
           id: log.id,
           apiName: (response.metadata[log.api] as LogsResponseMetadataApi).name,
           apiVersion: (response.metadata[log.api] as LogsResponseMetadataApi).version,
           method: log.method,
           status: log.status,
           timestamp: log.timestamp,
         })),
-      ),
+      })),
+      catchError(err => {
+        console.error(err);
+        return of({ data: [], error: err.message });
+      }),
```

Comment:
I think you shoudn't do this.
I've moved the catch error above for a good reason.
See https://github.com/gravitee-io/gravitee-api-management/pull/9265

Reply:
Gotcha.